esp/xslt/esdl2monitor.xslt

```diff
@@ -21,6 +21,7 @@
     <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
     <xsl:param name="responseType" select="''"/>
     <xsl:param name="requestType" select="''"/>
+    <xsl:param name="diffmode" select="'Monitor'"/>
```

Comment:
@rpastrana The tool treats 'Monitor' as the true default.  It runs it without specifying once to generate the monitor, and runs it specifying 'Compare' to generate a compare query.